### PR TITLE
Separate chain calls

### PR DESCRIPTION
**Pull request by sankaest**
| Q             | A
| ------------- | ---
| Readability fix  | according to https://github.com/yiisoft/docs/issues/155, chain calls must be put on separate lines.

### CODE REVIEW

**Comment by samdark** [comment]
Seems tests should be adjusted.

**Comment by sankaest** [comment]
Yep, I've seen it. But I can not understand, where it generate data for comparison. Will check it later.

**Comment by samdark** [comment]
Any idea about failing tests?

File: src/MigrationBuilder.php [hunk 1]
```diff
@@ -406,9 +463,9 @@ public function dropPrimaryKey(string $name, string $table): void
     public function addForeignKey(
         string $name,
         string $table,
-        $columns,
+               $columns,
```
[the remark this answers is not in the document]
Comment (sankaest) [reply src/MigrationBuilder.php]: IDE глюк. исправил.

**Pull request closed without merging**